web/src/containers/Login.js

```diff
@@ -38,21 +38,30 @@ class Login extends Component {
 
     let errorMessage = false;
     if (error === 'Unauthorized') {
-      errorMessage = 'The email or password you&apos;ve entered is incorrect.';
+      errorMessage = 'The email or password you’ve entered is incorrect.';
     } else if (error === 'Unauthorized') {
       errorMessage = 'Sorry! Something went wrong. Please try again.';
     }
 
     return (
       <Fragment>
         <Header />
-
         <CardForm
           title="Log in"
+          cancelable={false}
+          canSubmit={username.length && password.length}
           error={errorMessage}
           working={fetching}
           primaryButtonText={['Log in', 'Logging in']}
           onSave={this.handleSubmit}
+          footer={
+            <Fragment>
```

Comment:
Since the footer just renders a `div`, we should throw these in `p` tags.
```suggestion
            <p>
```